README.md

```diff
@@ -1,12 +1,10 @@
-[![JS.ORG](https://img.shields.io/badge/js.org-+-FFE70B.svg?style=flat-square)](http://js.org)
+[![js.org](https://img.shields.io/badge/js.org-+-FFE70B.svg?style=flat-square)](http://js.org)
 [![Donate](https://img.shields.io/badge/Donate-for_registrar_fees-1F87FF.svg?style=flat-square&logo=open-collective&logoColor=fff)](https://opencollective.com/js-org)
 
 ## Cleanup Scripts
-
```

Comment:
Please revert these line break removals. There should always be an extra linebreak between headings and the content.